src/Components/Web.JS/src/Platform/Circuits/ReconnectDisplay.ts

```diff
@@ -3,7 +3,7 @@
 
 export interface ReconnectDisplay {
   show(): void;
-  update(currentAttempt: number): void;
+  update(currentAttempt: number, secondsToNextAttempt: number): void;
```

Comment:
Is there anywhere else in Blazor's public API that we use seconds over milliseconds? I know that it's probably more detail than most `ReconnectDisplay` implementations would need, but I don't see the harm in giving a more precise value especially when more precision is supported by the underlying APIs. Plus, it would be more consistent.

Reply:
This function gets called every second, so I thought it would be better to provide a suitable display value rather than a precise one. It would be strange, for example, for the countdown to be 3.1, 2.1, 1.1...